bindings/python/examples/example.py

```diff
@@ -15,7 +15,7 @@
 
 EMPTY_ADDRESS = "atoi1qzt0nhsf38nh6rs4p6zs5knqp6psgha9wsv74uajqgjmwc75ugupx3y7x0r"
 client = iota_client.Client(
-    node=LOCAL_NODE_URL, node_sync_disabled=True)
+    nodes_name_password=[[LOCAL_NODE_URL]], node_sync_disabled=True)
```

Comment:
is this change right?

Reply:
For Python @bingyanglin decided with me that we only use a single field there, with which you can do the same stuff, so we don't need multiple fields